infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LocationTrackService.kt

```diff
@@ -288,10 +285,56 @@ class LocationTrackService(
     private fun getWithAlignmentInternal(version: RowVersion<LocationTrack>): Pair<LocationTrack, LayoutAlignment> =
         locationTrackWithAlignment(dao, alignmentDao, version)
 
-    fun getDuplicates(duplicateOf: IntId<LocationTrack>, publishType: PublishType): List<LocationTrackDuplicate> {
-        return dao.fetchDuplicates(duplicateOf, publishType)
+    @Transactional(readOnly=true)
+    fun getInfoboxExtras(publishType: PublishType, id: IntId<LocationTrack>,): LocationTrackInfoboxExtras? {
```

Comment:
En ole itse ihan varma `Infobox`-määreestä tyyppinimessä, sillä se viittaa frontin käyttöpaikkaan. Se on ihan ok jos tätä ei käytetä missään muualla bäkkärillä, mutta jos käytetään, niin saattaisin nimetä itse tämän jotenkin toisin.

Reply:
Joo tuntuu hiukan hassulta että backariin valuu frontin komponenttinimet, mutta samanaikaan tuntuu ihan fiksulta että tämä osoittaa suoraan siihen ainoaan käyttöpaikkaan - jos infoboxiin joutuu lisäämään jonkun lasketun tiedon niin tällä nimellä on aika selvää että sen lisääminen tänne ei vaikuta muihin paikkoihin.

Abstraktimpi nimi voisi olla vaikka getCalculatedFields tms, mutta enpäs osaa sanoa olisiko se parempi vai huonompi, joten ehkä tämä on ihan hyvä näin :shrug: